Menu/Provider/GroupMenuProvider.php

```diff
@@ -80,6 +103,15 @@ public function get($name, array $options = array())
                         'admin' => $admin,
                     );
                 } else {
+                    //NEXT_MAJOR: Remove if statement of null checker.
+                    if (null !== $this->checker) {
+                        if ((!empty($item['roles']) && !$this->checker->isGranted($item['roles']))
+                            || !$this->checker->isGranted($group['roles'], $item['route'])
```

Comment:
What if `$group['roles']` will be empty? Checking access rights should be omitted to not confuse users who starts with SonataAdmin and users that don't need this layer of security.